### PR TITLE
Fixed problem with running in quarantine mode (closes#236)

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\CC @inikulin, @AlexanderMoskovkin
PR

### CODE REVIEW

File: src/runner/browser-job.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ export default class BrowserJob extends EventEmitter {
         this.proxy             = proxy;
         this.browserConnection = browserConnection;
 
-        this.testRunQueue = tests.map(test => this._createTestRun(test, screenshots));
+        this.screenshots = screenshots;
```
Comment (inikulin) [inline src/runner/browser-job.js]: No need for the blank line before it

**Comment by inikulin** [comment]
\r-

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**